src/components/Header/Header.js

```diff
@@ -1,40 +1,40 @@
-import React, { useState } from "react";
-import { useDispatch } from "react-redux";
-import { Link } from "react-router-dom";
-import { FaAngleLeft } from "react-icons/fa";
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FaAngleLeft } from 'react-icons/fa';
 import {
   fetchAsyncMovies,
   fetchAsyncShows,
-} from "../../features/movies/movieSlice";
-import user from "../../images/user.png";
-import "./Header.scss";
+} from '../../redux/movies/movieSlice';
+import './Header.scss';
 
 const Header = () => {
-  const [ term, setTerm ] = useState('');
+  const [term, setTerm] = useState('');
   const dispatch = useDispatch();
   const submitHandler = (e) => {
     e.preventDefault();
     dispatch(fetchAsyncMovies(term));
     dispatch(fetchAsyncShows(term));
-    setTerm("");
-  }
+    setTerm('');
+  };
   return (
     <div className="header">
-        <div className="logo">
-         <Link to="/"><FaAngleLeft className="fa-icon"/></Link>
-        </div>
-        <div className="search-bar">
-          <form onSubmit={submitHandler}>
-            <input 
-            type='text' 
-            value={term} 
-            placeholder='search for movies' 
```

Comment:
- [Optional] This bar also appears on the detail page but we are not doing any search on this page.